electrum/gui/qt/main_window.py

```diff
@@ -2578,10 +2578,13 @@ def label(idx, ks):
             vbox.addWidget(ks_stack)
 
         vbox.addStretch(1)
-        btn_export_info = run_hook('wallet_info_buttons', self, dialog)
```

Comment:
same as in https://github.com/spesmilo/electrum/pull/7682/files - we need an ability to add multiple wallet info buttons